e2e-tests/helpers/format-currency.js

```diff
@@ -5,12 +5,14 @@ import { GBP_CURRENCY_CODE } from '../fixtures/currencies';
  * Transform a number into a currency string
  * @param {String} str: Amount
  * @param {String} Currency code - defaults to GBP
+ * @param {Number} decimalPlaces - defaults to 0
  * @returns {String} formatted currency
  */
-const formatCurrency = (str, currency = GBP_CURRENCY_CODE) => Number(str).toLocaleString('en', {
-  style: 'currency',
-  currency,
-  minimumFractionDigits: 0,
-});
+const formatCurrency = (str, currency = GBP_CURRENCY_CODE, decimalPlaces = 0) =>
```

Comment:
In the other instances of `formatCurrency`, the new param here is called `decimalPoints`, can we align?